### PR TITLE
Integrating assessment data into ERT

### PR DESCRIPTION
**Pull request by burzum619**

### CODE REVIEW

File: src/main/java/tds/exam/results/mappers/TestMapper.java
Comment (burzum619) [inline src/main/java/tds/exam/results/mappers/TestMapper.java]: Forgot to add this as a comment in the code, but this logic is ported over from ReportingDLL.TestGradeSpan_F(), line 1353

File: src/main/java/tds/exam/results/mappers/TestMapper.java
Comment (burzum619) [inline src/main/java/tds/exam/results/mappers/TestMapper.java]: Per @jtreuting , If the academic year is a range, select the more recent year.

File: src/main/java/tds/exam/results/mappers/TestMapper.java
Comment (millerts) [inline src/main/java/tds/exam/results/mappers/TestMapper.java:3]: nit - extra line breaks

File: src/main/java/tds/exam/results/mappers/TestMapper.java
Comment (millerts) [inline src/main/java/tds/exam/results/mappers/TestMapper.java]: Don't think it matters in the grand scheme of things but I usually use Long.valueOf instead.  This [answer](http://stackoverflow.com/questions/13145267/difference-between-long-valueofjava-lang-string-and-new-longjava-lang-string) has a good explanation

File: src/main/java/tds/exam/results/mappers/TestMapper.java
Comment (millerts) [reply src/main/java/tds/exam/results/mappers/TestMapper.java]: Can we add this as a Java comment.  Not that @jtreuting told you but explaining that the rule.

File: src/main/java/tds/exam/results/mappers/TestMapper.java
Comment (millerts) [reply src/main/java/tds/exam/results/mappers/TestMapper.java]: Assume you'll add it before the merge

File: src/main/java/tds/exam/results/mappers/TestMapper.java
Comment (millerts) [inline src/main/java/tds/exam/results/mappers/TestMapper.java]: Normally I wouldn't ask this but could you add a small JavaDoc explaining what the purpose of this method is?  Following the code I see that if it is a single grade we return "Third", but there are multiples and not high school we return something, then finally handle high school ranges. 

I think it is one off enough that a bit of java doc would help future maintainers

File: src/main/java/tds/exam/results/mappers/TestMapper.java
Comment (millerts) [inline src/main/java/tds/exam/results/mappers/TestMapper.java]: Can we remove this int and simply use `intGrades.size()` instead?  Since they will always be the same value.  Or if you would prefer to keep the int can we move it right above line 88 (`int numOfIntegerGrades = intGrades.size()`)?

**Review by millerts — APPROVED** [review]
minor comments

File: src/main/java/tds/exam/results/mappers/TestMapper.java
Comment (burzum619) [reply src/main/java/tds/exam/results/mappers/TestMapper.java]: No problem, it is a bit confusing at first glance

File: src/main/java/tds/exam/results/mappers/TestMapper.java
Comment (jtreuting) [reply src/main/java/tds/exam/results/mappers/TestMapper.java]: The rule is defined on page 4 of the TRT Data Model document: http://www.smarterapp.org/documents/TestResults-DataModel.pdf

"The school year in which the test is
administered. For academic years that
span a calendar year this is the four digit
year-end. E.g. 2013 for 2012-2013"

File: src/main/java/tds/exam/results/mappers/TestMapper.java
Comment (burzum619) [reply src/main/java/tds/exam/results/mappers/TestMapper.java]: Added this to the javadoc, but here's the rules:

     * Rules:
     *      If no grades are present, return an empty string
     *      If a single grade is present, return it
     *      If the grades is a contiguous range, hyphenate it (i.e. 7, 8, 9 =  7-9)
     *      If the grade range is a standard high school range (i.e. 9-12), return "HS"
     *      Otherwise, return the list of grades comma delimited, in ascending order.
     *      If non-numeric grades are present, they should be listed first

File: src/main/java/tds/exam/results/mappers/TestMapper.java
Comment (burzum619) [reply src/main/java/tds/exam/results/mappers/TestMapper.java]: yep, dunno how I missed that!

**Pull request merged**
